core/domain/app_feedback_report_constants_test.py

```diff
@@ -0,0 +1,33 @@
+# coding: utf-8
+#
+# Copyright 2022 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for core/domain/app_feedback_report_constants.py"""
+
+from __future__ import annotations
+
+from core.tests import test_utils
+from . import app_feedback_report_constants # pylint: disable=unused-import
```

Comment:
Why relative import? Also why is this needed?

Reply:
Ditto elsewhere.

Reply:
> Why relative import?
We use relative import to import the code file for which the associated test file is written for. We tend to keep both of them in the same folder and therefore relative import works well.

> Why is this needed?
Coverage tool accounts for a code file only when it is imported by a file ending with a suffix `_test.py`. To avoid the files sneaking past the coverage checks we need to import them in their associated test file. However, in this case, the file only has constants defined in it and no logic that should be tested; so, we simply import it and add a trivial test for it. This ensures that even if in future some logic is added to this file, it won't get past our coverage checks if it is not fully covered.